Makefile

```diff
@@ -344,6 +344,7 @@ install_prerequisites:
 	sudo apt-get install -y -q g++-s390x-linux-gnu || true
 	sudo apt-get install -y -q g++-riscv64-linux-gnu || true
 	sudo apt-get install -y -q g++ || true
+	[ -z "$(shell which python)" -a -n "$(shell which python3)" ] && sudo apt-get install -y -q python-is-python3 || true
```

Comment:
I don't have python-is-python3 package. But I have a python3 package. python3 looks like the package to install to get python3, no? What distro do you have? The rest of the install_prerequisites target is for a Debian/Ubuntu-like system.

Reply:
> What distro do you have? The rest of the install_prerequisites target is for a Debian/Ubuntu-like system.

I'm using Ubuntu 20.04 LTS (installed using ubuntu-20.04.1-live-server-amd64.iso).

> I don't have python-is-python3 package. But I have a python3 package. python3 looks like the package to install to get python3, no?

Since Python 2 already reached EOL, all scripts are expected to be ready for Python 3.
However, since I don't know whether all distributions are providing python3 as invocation name (e.g. /usr/bin/python3), I consider that explicitly providing a symlink named /usr/bin/python which points to python3 interpreter is safer than touching existing scripts.
The python-is-python3 package is there to provide such symlink.

Reply:
Okay, I don't what's the best course, but since this does not fail the target, I guess this is fine.